src/Viz.js

```diff
@@ -222,26 +204,26 @@ export default class Viz {
     _updateRootExpressionList () {
         this._rootExpressions = [
```

Comment:
Nice refactor of the properties!
Could we bring also some of that into this method, so we can avoid also the repetition here?